### ISSUE TITLE
Window resizing in Windows 11 is not responsive

### ISSUE THREAD

**Issue by birdybro** [Taiko2k/Tauon#979]
Just downloaded latest release 7.6.4. I haven't used the previous versions before on this computer. If resizing the window to a larger size, the responsiveness of the window is not working. It only is responsive upon shrinking the size of the window. See video for recreation:

https://user-images.githubusercontent.com/16388068/234920965-5fa1c0b1-623a-44ce-8717-e6ff1fc97592.mp4

**Comment by Solarunit** [comment]
I also see a weird glitches when resizing, but I am on Linux

**Comment by Taiko2k** [comment]
Yeah it does that on Windows. I'm pretty sure its an upstream bug with SDL so not much I can do about it. They are currently undergoing a major rewrite so not much point reporting it.

@Solarunit That might happen on x11 but it should be butter smooth with Wayland.

**Comment by Solarunit** [comment]
> That might happen on x11 but it should be butter smooth with Wayland.

oh, good to know

**Comment by C0rn3j** [comment]
@Solarunit Does this still repro today on SDL2 2.30.10?

They keep having releases while SDL3 stabilizes, so if it's still a problem there we should at least check that SDL3 does not suffer from it anymore - https://github.com/Taiko2k/Tauon/pull/1332

**Comment by Solarunit** [comment]
@C0rn3j 

tauon flatpak 7.8.3
Almost not an issue on Linux (x11), redrawing window not instant but pretty fast. Cursor doesn't change to resizing shape when you hover the bottom right corner btw.

**Comment by C0rn3j** [comment]
I can actually reproduce this on Linux with master, all you need is a large JXL image, go to the album art view and try resizing.

It's also visible on other album arts, but since JXL [does not currently do multithreading](https://github.com/Taiko2k/Tauon/issues/741#issuecomment-1330283280) in Tauon, it is extremely visible there.

It goes from being very smooth to rendering at 2 FPS at best.
